src/lib/packages/EditorContext/EditorContextProvider.tsx

```diff
@@ -48,6 +47,9 @@ export const EditorContextProvider: React.FC<EditorContextProviderProps> = ({
   debounceChange,
   defaultMarkdownValue,
 }) => {
+  const { mode } = useTextEditorContext();
+  const isEditable = useRef<boolean>(mode === 'active');
```

Comment:
Is it necessary to add type explicitly in this case?

Reply:
Fixed